rules/helm/package.tmpl.rb

```diff
@@ -0,0 +1,70 @@
+require "fileutils"
+require "open3"
+
+# Variables expanded by Bazel.
+package_dir = "[[package_dir]]"
+multipath_sep = "[[multipath_sep]]"
+tarsStr = "[[tars]]"
+generatedStr = "[[generated]]"
+helm = "[[helm]]"
+output_tgz = "[[output_tgz]]"
+
+# Variables expanded by gomplate.
+git_commit_short = '{{ (ds "workspace_status").STABLE_GIT_COMMIT_SHORT }}'
+git_branch = '{{ (ds "workspace_status").STABLE_GIT_BRANCH }}'
+
+# Create the temporary build directory for joining all the pieces required for packaging.
+tmp_build_dir = File.join("tmp", "build")
+build_dir = File.join(tmp_build_dir, package_dir)
+FileUtils.mkdir_p(build_dir)
+
+# Copy the source static files to the temporary build directory.
+Dir.glob(File.join(package_dir, "**", "*")) do |file|
+  dest = File.join(tmp_build_dir, file)
+  ftype = File.ftype(file)
+  if ftype == "directory"
+    FileUtils.mkdir_p(dest)
+  else
+    file = File.readlink(file) if ftype == "link"
+    FileUtils.cp(file, dest)
+  end
+end
+
+# Extract the tar files into the temporary build directory.
+tars = tarsStr.split(multipath_sep)
+tars.each do |file|
+  file = File.readlink(file) if File.ftype(file) == "link"
+  `tar xf "#{file}" -C "#{build_dir}"`
```

Comment:
Check exit status? Don't eat `STDOUT`, but show it to the user instead? (I prefer `Process.wait Process.spawn(…)` and never capture `STDOUT` to begin with.)